src/main/java/internet/shop/dao/jdbc/ProductDaoJdbcImpl.java

```diff
@@ -0,0 +1,109 @@
+package internet.shop.dao.jdbc;
+
+import internet.shop.dao.ProductDao;
+import internet.shop.exceptions.DataProcessingException;
+import internet.shop.lib.Dao;
+import internet.shop.model.Product;
+import internet.shop.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ProductDaoJdbcImpl implements ProductDao {
+    @Override
+    public Product create(Product item) {
+        String query = "INSERT INTO products (name, price) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query,
+                    Statement.RETURN_GENERATED_KEYS);
+            statement.setString(1, item.getName());
+            statement.setDouble(2, item.getPrice());
+            statement.execute();
+            ResultSet result = statement.getGeneratedKeys();
+            if (result.next()) {
+                item.setId(result.getLong(1));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Failed to create product "
+                    + item.getName(), e);
+        }
+        return item;
+    }
+
+    @Override
+    public Optional<Product> get(Long id) {
+        Product product = null;
+        String query = "SELECT * FROM products WHERE product_id = ? AND is_deleted = 0";
```

Comment:
```suggestion
        String query = "SELECT * FROM products WHERE product_id = ? AND is_deleted = FALSE";
```
this way it would easier to understand what's happening, change in all queries